crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -139,12 +162,40 @@ fn fragment(
         pbr_input.occlusion = occlusion;
 
         pbr_input.flags = mesh[in.instance_index].flags;
-
+#ifdef DEFERRED_PREPASS
+        pbr_functions::alpha_discard(pbr_bindings::material, output_color);
+        out.deferred = pbr_deferred_functions::deferred_gbuffer_from_pbr_input(pbr_input);
+        out.deferred_lighting_pass_id = pbr_bindings::material.deferred_lighting_pass_id;
+#ifdef NORMAL_PREPASS
+        out.normal = vec4(pbr_input.N * 0.5 + vec3(0.5), 1.0);
```

Comment:
In order to minimize the size of the deferred attachment we should not redundantly write the normals to the prepass normals attachment. Instead, we should have a pass after deferred that copies the normals from the gbuffer (using depth >= 1 with the deferred lighting pass id) as that should be a lot more efficient.

It's important that we don't read from the whole deferred gbuffer for things like SSAO where there's lots of samples because GPU memory is slow.